motionplan/tpspace/ptgCCS.go

```diff
@@ -0,0 +1,59 @@
+package tpspace
+
+import (
+	"math"
+
+	rutils "go.viam.com/rdk/utils"
+)
+
+// ptgDiffDriveCCS defines a PTG family combining the CC and CS trajectories, essentially executing the CC trajectory
+// and.
```

Comment:
trailing comment

Reply:
Whoops, good catch. Pushed the second half of the comment.